app/design/frontend/base/default/template/boltpay/popup.phtml

```diff
@@ -0,0 +1,117 @@
+<?php /* @var $this Mage_Core_Block_Template */?>
+
+<div id="bolt-popup" class="bolt-popup">
```

Comment:
can you inline`style="display: none"` here? I want to make sure it is never shown by accident